truenas/webui
# NAS-125806 / 24.04 / Disable restricted controls for Shares and Data Protection pages

**Testing**

1. Create a new local group on our server, with `READONLY` role and another group with `SHARING_MANAGER` role at **Credentials > Local Groups > Privileges**.

2. Log in with this user, and visit the **Shares** page, click through every button and clickable icon. 
   - **Expected result:** if action is illogical, or corresponding API call results in an error, then this action should not be available in WebUI for the current user

5. Do the same for **Data Protection** page.
